data/src/test/java/org/apache/iceberg/data/orc/TestOrcRowIterator.java

```diff
@@ -74,7 +75,7 @@ public void writeFile() throws IOException {
             .schema(DATA_SCHEMA)
             // write in such a way that the file contains 2 stripes each with 4 row groups of 1000
             // rows
-            .set("iceberg.orc.vectorbatch.size", "1000")
+            .set(TableProperties.ORC_WRITE_BATCH_SIZE, "1000")
```

Comment:
Is this change needed? It looks unrelated to the deprecation changes.

Reply:
Or is this a different property? I see the matching constant was removed.

Reply:
`iceberg.orc.vectorbatch.size` was deprecated for quite some time. I've restored all code that removes deprecated properties and code that deals with legacy properties